_layouts/post.html

```diff
@@ -14,15 +14,17 @@
 
   <main id="main-content" class="main-content">
```

Comment:
I know we're not using `position: absolute;` anymore for the component page, which was having that `display: inline-block` issue where the menu wasn't clickable on firefox.

It could be the time to reevaluate if we need positioning on it at all. I say this because if you inspect `.main` on the pages w/ 2 columns you see this:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/3385219/70152772-6df45280-1673-11ea-8aa0-0048a913fbe8.png">

The code:
```css
//
// _uswds-theme-custom-styles.scss:470
//
.main-content {
  @include u-pin("right");
  @include u-pin("bottom");
  padding-top: units(5); // starting: not scrolled
  position: relative;
  width: 100%;

  @include at-media("desktop") {
    width: calc(100% - #{units($width-nav-sidebar)});
  }

  .lt-ie9 & {
    width: 75%;
  }
}
```